src/Entity/User.php

```diff
@@ -894,4 +897,14 @@ public function canUpdateUser(User $actor): bool
             return $this->apDomain === $actor->apDomain;
         }
     }
+
+    public function getRelatedLinks(): array
+    {
+        return $this->relatedLinks;
+    }
+
+    public function setRelatedLinks(array $relatedLinks): void
```

Comment:
This one is actually not used. Maybe make the variable private and use this one?

Reply:
deleted